ledger/catchpointtracker.go

```diff
@@ -413,7 +424,10 @@ func (ct *catchpointTracker) newBlock(blk bookkeeping.Block, delta ledgercore.St
 // number that can be removed from the blocks database as well as the lookback that this
 // tracker maintains.
 func (ct *catchpointTracker) committedUpTo(rnd basics.Round) (retRound, lookback basics.Round) {
-	return rnd, basics.Round(0)
+	ct.catchpointsMu.RLock()
+	defer ct.catchpointsMu.RUnlock()
+	retRound = ct.cachedDBRound
+	return retRound, basics.Round(0)
```

Comment:
why do we always return basics.Round(0) in this function? Not your change, just curious.

Reply:
because catchpoint tracker does not really care about deltas and and in-memory lookback. It was not caring about the min block db round to maintain before this change as well.